tests/test_cli.py

```diff
@@ -52,6 +52,7 @@ def test_cli_single_input(self):
             elif test.inputformat == "tsv":
                 # These test only run on TSV inputs
                 self.run_convert(runner, test)
+                self.run_convert(runner, test, "ontoportal_json")
```

Comment:
Add ontoportal output to CLI tests

Context:
I made this suggestion while working on FHIR outputs, https://github.com/mapping-commons/sssom-py/pull/285#discussion_r1590199740, but I decided to split the suggestion into this new PR